src/components/core/menu/mainMenu/MainMenuLeft.tsx

```diff
@@ -16,20 +21,24 @@ export const MainMenuLeft: FC = () => {
       data-testid="main-nav"
     >
       <Link
-        to={PathNames.strategies}
+        to="/"
         onClick={() => carbonEvents.navigation.navHomeClick(undefined)}
       >
         <LogoCarbon className={'w-34'} />
       </Link>
 
       <div className={'hidden space-x-24 md:block'}>
-        {menuItems.map(({ label, href, hrefMatches }, index) => {
-          const isSamePage = isPathnameMatch(pathname, href, hrefMatches);
+        {menuItems.map(({ label, href }, index) => {
+          let isSamePage = isSamePageLink(href);
```

Comment:
You can use `const` now
```suggestion
          const isSamePage = isSamePageLink(href);
```